.github/workflows/build-push-images.yml

```
@@ -0,0 +1,28 @@
+name: Build and push images
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build_push:
+    runs-on: ubuntu-latest
+    env:
```

actionlint (1.7.7)

expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)